src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -94,13 +94,32 @@
         {%- set attr = attr|merge({class: (attr.class|default('') ~ ' form-inline')|trim}) -%}
         <div {{ block('widget_container_attributes') }}>
             {{- form_errors(form) -}}
-            {%- if with_years %}{{ form_widget(form.years) }}{% endif -%}
-            {%- if with_months %}{{ form_widget(form.months) }}{% endif -%}
-            {%- if with_weeks %}{{ form_widget(form.weeks) }}{% endif -%}
-            {%- if with_days %}{{ form_widget(form.days) }}{% endif -%}
-            {%- if with_hours %}{{ form_widget(form.hours) }}{% endif -%}
-            {%- if with_minutes %}{{ form_widget(form.minutes) }}{% endif -%}
-            {%- if with_seconds %}{{ form_widget(form.seconds) }}{% endif -%}
+            <div class="table-responsive">
+                <table class="table table-bordered table-condensed table-striped" style="width: auto;">
```

Comment:
`style="width: auto;` should be removed I guess, however the default bootstrap `.table` class adds a `width: 100%` rule. But we should probably don't care and let the user apply some additional style?